src/core/sio1.h

```diff
@@ -19,18 +19,30 @@
 
 #pragma once
 
+#include <stdint.h>
+
 #include <string>
 
 #include "core/psxemulator.h"
 #include "core/psxmem.h"
 #include "core/r3000a.h"
 #include "core/sio1-server.h"
 #include "core/sstate.h"
+#include "support/file.h"
 
-//#define SIO1_CYCLES (m_baudReg * 8)
+//#define SIO1_CYCLES (m_regs.baud * 8)
 #define SIO1_CYCLES (1)
 
 namespace PCSX {
+
+struct sio1Registers {
```

Comment:
Please capitalize types.

```suggestion
struct SIO1Registers {
```